src/sdp.ts

```diff
@@ -76,7 +86,9 @@ export function toSupportedHashFunction (name: multihashes.HashName): string {
   }
 }
 
-// Convert a multiaddr into a SDP
+/**
+ * Convert a multiaddr into a SDP
```

Comment:
Might be nice to write fully what `SDP` means as `<full name> (SDP)`, small effort but a cognitive ease of mind